### DISCUSSION TITLE
More control over page display/timing

### DISCUSSION THREAD

**Discussion by ireneybean** [gickowtf/pixoo-homeassistant#23] [Ideas]
Hi! I'm just getting started using this integration, and it's working very well for me. I work from home and I've worked up a very basic little screen that shows my microphone/camera status so that if any of my family walk into the room they can immediately see whether or not they're being broadcast to all my colleagues or not. That's working great for me.

I've also got my page 2 set to my current favorite one of the build in clocks, and the screens swap on an interval, as documented. What I'd REALLY like is to be able to have a little more control over the switching. For example, if I'm not in the room and my computer isn't active, the microphone/camera screen really doesn't need to show, but I'd like to still have the clock show.  I think there are many ways this could be accomplished, and maybe there's even something there already and I'm just not seeing it, but  I thought I'd throw the idea out there. 

I'm definitely willing to help out too (new to ha, but I think I can muddle through most of it), just let me know your thoughts!

**Comment by gickowtf** [comment]
Hey, one idea might be to put a door/window sensor on your door and configure it as a trigger. This way, when someone opens the door, you can indicate via push on your Pixoo64 that your microphone is active.

Otherwise, I am of course always open to any ideas and help. Thank you very much for your offer. You can make pull requests at any time.

**Reply by ireneybean** [reply]
Ah right, that makes sense. I'll give that a shot, thanks.
